_articles/account/export-your-data.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: article
+title: Export your data
```

Comment:
Do you think it would be worth specifying that this is an article for exporting bitwarden data? Or is this too verbose?